pom.xml

```diff
@@ -68,6 +68,7 @@
     <revision>2.5.7</revision>
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.138.4</jenkins.version>
+    <slf4jVersion>1.7.26</slf4jVersion>
```

Comment:
This silences the enforcer, but it isn't the ideal way to fix the issue in Jenkins. `slf4j` and associated packages are dependencies of Jenkins core itself, so the version that the plugin specifies here is not actually used in practice (see [this page](https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/) for more information). The best way to fix this is to update the parent POM to a 4.x version, which uses a BOM to address conflicts with dependencies of Jenkins core itself, and then update `jenkins.version` here to a version that supports the BOM, e.g. `2.176.4`. After that, you will be able to remove this line, and `1.7.25` will automatically be used when running against the version of Jenkins and `1.7.26` will be used when running against the new version of Jenkins, and the enforcer will be happy.

Reply:
It was my first tentative in other PR and I removed the jenkins.version to leave the organisation pom guide the jenkins version.
Do you prefer set a specific version of jenkins instead use that one defined on organisation pom?

Reply:
Any specific reason why prefer 2.176.4 instead of that one defined in organisation pom (2.204) ?

Reply:
With 2.176.4 test fails

Reply:
With 2.222.3 test fails.

Override BOM plugin version with those defined in master test fails again.
Single test in eclipse not happens, debug in mvn clean test -Dmave.surefure.debug step by step not happens only run when run all together not in debug.
Seems a timing issue. The test look if a FreestyleProject master exists for top/stuff but it has not yet been by organisation folder scan.

Reply:
2.176.4 is a conservative choice. 
Not sure what do about test failures.

Reply:
The problem is that does not wok with nothing.
The organisation pom version is the cause <= 3.56 test passes. With version 3.57 fails
Using jenkins.version 2.176.4 test passes (locally) but will fail on CI system because runs with 2.222.3 and will fail on enforcer.